plugins/destination/bigquery/client/client_test.go

```diff
@@ -40,7 +41,8 @@ func TestPlugin(t *testing.T) {
 		plugin.WithTestDataOptions(schema.TestSourceOptions{
 			SkipMaps: true,
 			// https://github.com/cloudquery/cloudquery/issues/12022
-			SkipTimes: true,
+			SkipTimes:     true,
```

Comment:
If we are skipping times, what's the reason we need to configure the precision?

Reply:
time32 & time64 only, not timestamp